SharedPackages/UIComponents/Sources/UIComponents/macOS/CustomToggleControl.swift

```diff
@@ -104,6 +115,100 @@ public final class CustomToggleControl: NSControl {
     private var animationTargetProgress: CGFloat = 0.0
     private let animationDuration: CFTimeInterval = 0.15
 
+    private var leftSegmentToolTip: String?
```

Comment:
I did a refactor on this component to be more similar to NSSegmentedControl. No changes other than the public API